test/CMakeLists.txt

```diff
@@ -63,7 +63,10 @@ add_subdirectory(files)
 add_subdirectory(util)
 
 if(NGRAPH_CPU_ENABLE)
-    set(SRC ${SRC} backend_performance.cpp codegen.cpp cpu_fusion.cpp cpu_test.cpp)
+    set(SRC ${SRC} backend_performance.cpp cpu_fusion.cpp cpu_test.cpp)
+    if (NOT NGRAPH_DEX_ONLY)
```

Comment:
codegen has nothing to do with DEX. There is more than just the CPU backend. Leave this test in.